code/datums/diseases/parrotpossession.dm

```diff
@@ -25,7 +25,7 @@
 		cure()
 		return FALSE
 
-	if(length(parrot.speech_buffer) && prob(parrot.speak_chance))
+	if(length(parrot.speech_buffer) && DT_PROB(parrot.speak_chance, delta_time))	// I'm not going to dive into polycode trying to adjust that probability. Enjoy doubled ghost parrot speach
```

Comment:
Didn't you need to do this anyway? Also can't you just do the * 0.5 trick?

Reply:
Multiplying by 0.5 only works with small probabilities, and even then it's only mostly accurate. Also, speak chance is handled by the npc subsystem and I'm already overhauling SSmobs time handling. I don't want to do SSnpcpool too.